### PR TITLE
fix(dashboard): fix invalid javascript on standalone panels

### PR DESCRIPTION
**Pull request by MattMcNam**

### CODE REVIEW

**Comment by Lange** [comment]
The tests all failed for a pretty strange reason:

```
bower install
bower ENOENT        No bower.json present
```

`bower.json` is definitely still in the repo, so I'm not sure what's going on.

**Comment by MattMcNam** [comment]
Added unit test, and a check for autodeps being disabled. Causes mocha to hang if disabled here.

All passing locally, I'll see if Travis works right this time

**Comment by Lange** [comment]
My hunch is that this is a problem with travis. All our builds are failing now, but I had builds passing just this morning.

File: test/setup.js [hunk 1]
```diff
@@ -16,6 +16,11 @@ const C = require('./setup/test-constants');
 before(function (done) {
 	this.timeout(0);
 
+	if (C.CONFIG.autodeps && (C.CONFIG.autodeps.npm === false || C.CONFIG.autodeps.bower === false)) {
+		throw new Error('Autodeps disabled! ' +
+			'Please enable auto-installing dpenedencies cfg/nodecg.json before running tests');
```
Comment (Lange) [inline test/setup.js]: "dependencies" is misspelled

**Comment by MattMcNam** [comment]
Good catch, fixed

**Comment by Lange** [comment]
Tests are green, commits look good, merge at your leisure.

**Pull request merged**